core/src/main/java/org/arquillian/cube/q/core/InstallProxy.java

```diff
@@ -26,15 +26,17 @@ public void install(@Observes(precedence = 100) CubeDockerConfiguration configur
         
         DockerCompositions cubes = configuration.getDockerContainersContent();
         ProxyManager installer = serviceLoaderInst.get().onlyOne(ProxyManager.class);
-        
-        Proxy proxy = installer.install(cubes);
-        proxyInst.set(proxy);
 
-        final CubeContainer cube = proxy.getCube();
-        cubes.add(proxy.getName(), cube);
+        if (installer != null) {
+            Proxy proxy = installer.install(cubes);
+            proxyInst.set(proxy);
+
+            final CubeContainer cube = proxy.getCube();
+            cubes.add(proxy.getName(), cube);
 
-        System.out.println("PROXY INSTALLED");
-        System.out.println(ConfigUtil.dump(cubes));
+            System.out.println("PROXY INSTALLED");
```

Comment:
Yes I thought the same, but then you have the problem of missing important information because of wrong logger (for example not having the correct logger jar ) For this reason in Arquillian Governor, I think that Chamaleon and Cube we started to print configuration information in sysout. It is not perfect I know but at least it is effective.